packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -79,7 +79,7 @@ export const PRESET_METADATA = [
 	},
 	{
 		path: [ 'spacing', 'spacingSizes' ],
-		valueKey: 'spacingSizes',
+		valueKey: 'size',
```

Comment:
It looks like `valueKey` is used in global styles to point to the key to be used to retrieve the value for a particular preset. So in this case it should be `size`. Let me know if anyone knows if it needs to be set to `spacingSizes` for any other use cases, though!